client/src/Pages/Game.js

```diff
@@ -43,6 +53,23 @@ export default function GamePage({
       wordData[randomNumber].germanNoun;
     return setSelectedWord(wordSelected);
   }
+  function timeIsOut() {
+    const index = wordData.findIndex(
+      (word) => word.germanNoun === selectedWord
+    );
+    if (counter === 0) {
+      setCounter(null);
+      const correctAnswer =
+        wordData[index].gender;
+      setCorrectAnswer(correctAnswer);
+      setIsAnswerDisplayed(false);
+      setPlayerLives(1);
+      setTimeout(() => {
+        setIsAnswerDisplayed(true);
+        randomWord();
+      }, 2000);
+    }
+  }
 
   function handleClick(nounGender) {
```

Comment:
The function name `handleClick` is very generic considering how much logic you have in this function. Seems like a big part of your game logic, so try to find a name that represents what is happening within the function